core/amber/src/main/resources/python_udf/topic_modeling_trainer.py

```diff
@@ -1,14 +1,18 @@
+import logging
+
 import gensim
 import gensim.corpora as corpora
 import pandas
+from loguru import logger
```

Comment:
why do we have to import logger here but we have removed it from all the other files? Maybe this is related to the intercept class that you have in the main file which intercepts exceptions? Please add comments in that file to make it clear